x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -219,26 +219,7 @@ export function TimeseriesChart({
           tickFormat={yTickFormat ? yTickFormat : yLabelFormat}
           labelFormat={yLabelFormat}
         />
-
-        {showAnnotations && (
-          <LineAnnotation
-            id="annotations"
-            domainType={AnnotationDomainType.XDomain}
-            dataValues={annotations.map((annotation) => ({
-              dataValue: annotation['@timestamp'],
-              header: asAbsoluteDateTime(annotation['@timestamp']),
-              details: `${i18n.translate('xpack.apm.chart.annotation.version', {
-                defaultMessage: 'Version',
-              })} ${annotation.text}`,
-            }))}
-            style={{
-              line: { strokeWidth: 1, stroke: annotationColor, opacity: 1 },
-            }}
-            marker={<EuiIcon type="dot" color={annotationColor} />}
-            markerPosition={Position.Top}
-          />
-        )}
```

Comment:
I can't quite wrap my head around whether moving this to the context works out. But it's probably fine.